zanata-war/src/main/webapp/WEB-INF/layout/language/request_join_modal.xhtml

```diff
@@ -47,35 +47,35 @@
       <div class="modal__content l--pad-all-1">
         <div class="form__item" id="roles">
           <span class="l--push-bottom-quarter">#{msgs['jsf.RequestRoleAs']}</span>
-          <s:fragment rendered="#{!languageJoinAction.isTranslator()}">
+          <ui:fragment rendered="#{!languageJoinAction.isTranslator()}">
             <zanata:checkbox id="translatorChk" value="translator"
               checked="#{languageJoinAction.requestAsTranslator}"
               onValueChanged="bindRole" label="#{msgs['jsf.Translator']}"/>
-          </s:fragment>
+          </ui:fragment>
 
-          <s:fragment rendered="#{!languageJoinAction.isReviewer()}">
+          <ui:fragment rendered="#{!languageJoinAction.isReviewer()}">
             <zanata:checkbox value="reviewer" checked="#{languageJoinAction.requestAsReviewer}"
               onValueChanged="bindRole" label="#{msgs['jsf.Reviewer']}"/>
-          </s:fragment>
+          </ui:fragment>
 
-          <s:fragment rendered="#{!languageJoinAction.isCoordinator()}">
+          <ui:fragment rendered="#{!languageJoinAction.isCoordinator()}">
             <zanata:checkbox value="coordinator" checked="#{languageJoinAction.requestAsCoordinator}"
               onValueChanged="bindRole" label="#{msgs['jsf.Coordinator']}"/>
-          </s:fragment>
+          </ui:fragment>
         </div>
 
-        <s:decorate id="messageField" template="/WEB-INF/layout/field.xhtml">
+        <zanata:decorate id="message" componentId="messageField">
```

Comment:
componentId need to match the render value inside ```<a4j:ajax event="blur" render="messageField" execute="@this"/>```. I tried to minimize changes so reused the old id here. Same with other places